code/_macros.dm

```diff
@@ -1,6 +1,8 @@
 #define Clamp(x, y, z) 	(x <= y ? y : (x >= z ? z : x))
 #define CLAMP01(x) 		(Clamp(x, 0, 1))
 
+#define span(class, text) ("<span class='[class]'>[text]</span>")

```

Comment:
It'll replace a call to it with "string".

So `src << span("a", "thing")` will result in `src << "<span class='[class]'>[text]</span>"`. The much larger issue may arise from someone doing the other stupid, like this: `src << "[span()]"`